README.md

```diff
@@ -43,7 +43,7 @@ e.g.
 
 ```
 $ docker build -f ./Dockerfile.dev -t charty-dev:0.1 .
-$ docker run -it charty-dev:0.1 irb -rcharty
+$ docker run -it -v ${PWD}:/charty -w /charty charty-dev:0.1 ./bin/console
```

Comment:
I think we sometimes need to run `bundle install` before running `bin/console`.

Reply:
You are quite right.  I will add `$ bundle install` to the head.

Reply:
Fixed it. https://github.com/red-data-tools/charty/commit/b57b0aedf12c427f80191f15777b5586e1a99ea9